tests/unit/plugins/test_colcon.py

```diff
@@ -1053,6 +1081,14 @@ def __eq__(self, args):
                             ".*--ament-cmake-args.*{}".format(re.escape(expected_args))
                         ),
                     )
+                if colcon_packages_ignore:
+                    expected_args = "".join(colcon_packages_ignore)
```

Comment:
These are space-separated, are they not?

Reply:
yeah you are right I will fix it